pkg/providers/imagefamily/bootstrap/aksbootstrap.go

```diff
@@ -40,7 +40,7 @@ type AKS struct {
 	Arch                           string
 	TenantID                       string
 	SubscriptionID                 string
-	UserAssignedIdentityID         string
+	KubeletIdentityClientID        string
```

Comment:
great name change!

Reply:
Do we worry about these values being different from those in azure json? So people may grep for UserAssignedIdentityID since its called that everywhere else?

Reply:
(discussed offline)

It is better this way given that the `UserAssignedIdentityID` that is known by `azure.json` is in fact `KubeletIdentityClientID` that is known by the cluster. This confusion is the prime motivation for this entire work.

Currently, I choose to keep this variable name until we feed it to the bootstrap module. There could be even more opportunities around that area to make it even more clear...

Reply:
Now that we have this knowledge, we should be aware when coming across their uses in those "everywhere else." Maybe we can catch some big bugs.